sam-karis/airtech
# #Feat add flights

## What does this PR do?
------
It adds flights creation by admin and view for all functionality

## Description of Task to be completed?
-----
- add flight model, serializers, URLs, and views
- update test

## How should this be manually tested?
------
clone the repo
check out to `ft-add-flight-167660073`
add environment variables as defined in [`.env_example`](https://github.com/sam-karis/airtech/blob/develop/.env_example)
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[add flights](https://www.pivotaltracker.com/story/show/167660073)


## Screenshots (if appropriate)
------
#### admin dashboard creating flight
![image](https://user-images.githubusercontent.com/22454909/63218545-cd326380-c165-11e9-913c-3c2a760a863b.png)

#### view flights
![image](https://user-images.githubusercontent.com/22454909/63218554-f05d1300-c165-11e9-9f44-941ca5a10031.png)
